### PR TITLE
[full-ci] Remove config options that ain't available in web anymore

### PR DESCRIPTION
**Pull request by AlexAndBear**
## Related Issue

- Fixes <issue_link>

## How Has This Been Tested?

- test environment:
- test case 1:
- test case 2:
- ...

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Technical debt
- [ ] Tests only (no source changes)

## Checklist:

- [ ] Code changes
- [ ] Unit tests added
- [ ] Acceptance tests added
- [ ] Documentation ticket raised: <link>

### CODE REVIEW

File: changelog/unreleased/remove-unavailable-web-config-options.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+Change: remove unavailable web config options
+
+We've removed config options from the web package, that are no longer available in web.
```
Comment (kulmann) [inline changelog/unreleased/remove-unavailable-web-config-options.md:3]: please add something like `Please check the web changelog for more details.`

**Comment by ScharfViktor** [comment]
Screenshot 2024-07-23 at 16 01 37

I guess we need https://github.com/owncloud/web/pull/11251/commits/89f3cb0f4ee2b98f73e818546295cf7bc557ad50

**Comment by ScharfViktor** [comment]
> @ScharfViktor I have no idea how to bring it in, do we need a new web release ? :O

Let's see what CI has to say. If we have failed tests again, we'll need to make a new web release

**Comment by ScharfViktor** [comment]
now this test is green https://drone.owncloud.com/owncloud/ocis/37429/55/9

File: services/web/pkg/config/defaults/defaultconfig.go [hunk 2]
```diff
@@ -101,38 +101,23 @@ func DefaultConfig() *config.Config {
...
-						ID:   "preview",
-						Path: "web-app-preview",
-						Config: map[string]interface{}{
-							"mimeTypes": []string{
-								"image/tiff",
-								"image/bmp",
-								"image/x-ms-bmp",
-							},
-						},
+						ID:     "preview",
```
Comment (kulmann) [inline services/web/pkg/config/defaults/defaultconfig.go:104]: Now with the mime types config being removed you can remove the preview app from the external apps entirely and add `preview` to the `Apps` slice in line 101. We only had it as external app to be able to pass config.

**Review by kulmann — CHANGES_REQUESTED** [review]

**Review by kulmann — APPROVED** [review]
thank you 🥇

**Pull request merged**